Library/Homebrew/formula_auditor.rb

```diff
@@ -971,5 +949,43 @@ def linux_only_gcc_dep?(formula)
 
       true
     end
+
+    def collect_previous_info
```

Comment:
```suggestion
    def get_previous_versions_commits
```

or something

Reply:
Fwiw, we have to avoid the `get_` prefix because of the `Naming/AccessorMethodName` RuboCop: `Do not prefix reader method names with get_.`

`get_previous_versions_commits` doesn't convey the same purpose to me, as the goal of the method is to extract information from the commits, not just fetch the commits themselves.

If we rework the method to return the instance variables in an array (i.e., `[@previous_committed, @newest_committed]`) as mentioned in another comment, how about `committed_version_info`? `previous_committed_version_info` would be less ambiguous but also a bit lengthy.